Docs/packer seo templates batch1 (#13282)
hashicorp/packer: website/content/docs/templates/hcl_templates/syntax.mdx

---
page_title: HCL syntax reference
description: |-

>   The HCL syntax combines declarative structure with expressions into Packer templates. Learn about the HCL syntax for defining infrastructure as code. 

Reviewer: ```suggestion
  HCL syntax combines declarative structure with expressions into Packer templates. Learn how to use HCL syntax to define infrastructure as code. 
```
